src/_variables.scss

```diff
@@ -1,8 +1,8 @@
 @import '~box-react-ui/lib/styles/variables';
 
-$mobile: '(max-width: 768px)';
-$tablet: '(min-width: 768px) and (max-width: 1024px)';
-$desktop: '(min-width: 1025px)';
+$mobile: '(max-width: 767px)';
+$tablet: '(min-width: 767px) and (max-width: 1024px)';
```

Comment:
These values are inclusive, so rules will overlap. Is that intended?

Reply:
yup